locales/en.default.schema.json

```diff
@@ -448,7 +448,7 @@
       "name": "Announcement bar",
       "settings": {
         "auto_rotate": {
-          "label": "Auto-rotate announcements"
+          "label": "Auto-rotate announcements on desktop"
```

Comment:
FWIW I kind of find the "announcements" part to be a little redundant under an "Announcements" heading anyway. "Auto-rotate on desktop" seems sufficient to me and would probably keep it to one line in english.

Reply:
Agreed - let's remove "announcements" from the checkbox copy here given the heading.

Reply:
Side note: I think the `Change slides every `was probably taken directly from the slideshow setting. Since these aren’t slides, and since we have the announcements heading, I think we could remove the word `slides` there too.